test/chi/components/date-picker.pug

```diff
@@ -242,9 +247,18 @@ script.
           min: '01/24/2019'
         }
       );
+      chi.datePicker(
+        document.getElementById('datepicker-3'),
+        {
+          date: '01/24/2019',
+          min: '01/24/2019'
+        }
+      );
 
       document.getElementById('datepicker-1').focus();
       document.getElementById('datepicker-2').focus();
+      document.getElementById('datepicker-3').focus();
+      document.getElementById('datepicker-3').blur();
```

Comment:
Why focus and then blur back?

Reply:
Also, please, resolve the conflicts and run `chi build` as I'm not seeing integrity SHAs to be updated

Reply:
Because otherwise the cursor is left there and the test image fails, with blur it just leaves it unfocused, and ok!